server/routes/deleteThread.ts

```diff
@@ -3,24 +3,33 @@ import { factory, formatFilename } from '../../shared/logging';
 const log = factory.getLogger(formatFilename(__filename));
 
 const deleteThread = async (models, req: Request, res: Response, next: NextFunction) => {
+  const { thread_id } = req.body;
   if (!req.user) {
     return next(new Error('Not logged in'));
   }
-  if (!req.body.thread_id) {
+  if (!thread_id) {
     return next(new Error('Must provide thread_id'));
```

Comment:
Would be good to add Errors constant to remove hardcoded errors.

Reply:
Fixed